packages/amplication-nest-logger-module/src/logger.module.ts

```diff
@@ -0,0 +1,17 @@
+import { DynamicModule, Module } from "@nestjs/common";
+import { WinstonModule } from "nest-winston";
+import { defaultMeta, winstonConfigFactory } from "./config.factory";
+
+export type ModuleOptions = defaultMeta;
+
+@Module({})
+export class AmplicationLoggerModule {
+  static register(options: ModuleOptions): DynamicModule {
```

Comment:
```suggestion
import { LoggerMetadata, winstonConfigFactory } from "./config.factory";

export interface AmplicationLoggerModuleOptions {
  metadata: LoggerMetadata;
}

@Module({})
export class AmplicationLoggerModule {
  static register(options: AmplicationLoggerModuleOptions): DynamicModule {
```
Let's rename some stuff here to be more specific. `ModuleOptions` is not a good name for autocompletion and will not make developers' life easier. It is very likely to clash with other modules.

As I mentioned above, this is a use case for an `interface` and far from a `type` use case.

Also, let's use `metadata` as an options property and not as the root of the AmplicationLoggerModule. Otherwise, we limit ourselves to the Winston API and cannot deviate from it.